modules/releng.gradle

```diff
@@ -343,6 +344,80 @@ prepareMajorIncrement {
 	}
 }
 
+tagAppReleases {
+	description = "Gathers the Git IDs from all app changelog files and tags them as 'marketplace-appName-appVersion'."
+
+	doLast {
+		FileTree appChangelogFileTree = fileTree(dir: projectDir, includes: [".releng/apps/**/app.changelog"])
+
+		appChangelogFileTree.each {
+			appChangelog ->
+
+			String appName = appChangelog.getParentFile().getName()
```

Comment:
In Gradle/Groovy, you can use properties. So it's `appChangelogFile.parentFile.name`.